assembl/static2/js/app/utils/globalFunctions.js

```diff
@@ -296,5 +298,13 @@ export function getIconPath(icon: string, color: string = '') {
   return color ? `${ICONS_PATH}/${color}/${icon}` : `${ICONS_PATH}/${icon}`;
 }
 
+
 // We `pictureId + 1` because there is no image in the S3 bucket with 0 as an id
-export const getPictureUrl = (pictureId: number) => `${PICTURE_BASE_URL}${pictureId + 1}${PICTURE_EXTENSION}`;
\ No newline at end of file
+export const getPictureUrl = (pictureId: number) => `${PICTURE_BASE_URL}${pictureId + 1}${PICTURE_EXTENSION}`;
+
+export const getRandomPictureUrl = () => {
```

Comment:
This function should be deleted. I think it comes from a rebase. Only getPictureUrl should be kept.

Reply:
Indeed, must have crept in from a rebase. I'll remove it